app/api/v1/reports/models.py

```diff
@@ -1,23 +1,24 @@
-reports = []
-total_reports_created = 1
-
 class ReportsModel:
-    def __init__(self):
-        self.db = reports
-        self.total_reports_created = total_reports_created
+    reports = []
```

Comment:
I think a better way to do this would be to have this in a separate class that holds both records and users globally. Or just retaining it as a global variable